OSX show-device.js throws error

After downloading and installing node to get npm the 'npm install' command ran only raising a couple of warnings...

  CC(target) Release/obj.target/hidapi/hidapi/mac/hid.o
../hidapi/mac/hid.c:255:20: warning: comparison of integers of different signs:
      'CFIndex' (aka 'long') and 'size_t' (aka 'unsigned long') [-Wsign-compare]
                if (chars_copied == len)
                    ~~~~~~~~~~~~ ^  ~~~
../hidapi/mac/hid.c:295:20: warning: comparison of integers of different signs:
      'CFIndex' (aka 'long') and 'size_t' (aka 'unsigned long') [-Wsign-compare]
                if (used_buf_len == len)
                    ~~~~~~~~~~~~ ^  ~~~
2 warnings generated.

When I run 'node show-devices.js' I get the following error...

Quorra:src adam$ node show-devices.js 

module.js:340
    throw err;
          ^
Error: Cannot find module 'node-hid'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/Users/adam/Documents/Development/Paraglider-Sandbox/node-hid/src/show-devices.js:1:73)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
Quorra:src adam$ 

Have I missed a step in the installation?

--- a/src/buzzers.js
+++ b/src/buzzers.js
@@ -2,7 +2,7 @@
 
 var util = require('util');
 var events = require('events');
-var HID = require('node-hid');
+var HID = require('../');
 
 // buzzer protocol info: http://www.developerfusion.com/article/84338/making-usb-c-friendly/
 

--- a/src/powermate.js
+++ b/src/powermate.js
@@ -6,7 +6,7 @@
 // to the PowerMate contains zero in the first byte and the brightness
 // of the LED in the second byte.
 
-var HID = require('node-hid');
+var HID = require('../');
 var util = require('util');
 var events = require('events');
 

--- a/src/show-devices.js
+++ b/src/show-devices.js
@@ -1,3 +1,3 @@
-var HID = require('node-hid');
+var HID = require('../');
 
 console.log('devices:', HID.devices());

--- a/src/test-ps3.js
+++ b/src/test-ps3.js
@@ -1,4 +1,4 @@
-var HID = require('node-hid');
+var HID = require('../');
 var REPL = require('repl');
 
 var repl = REPL.start('node-hid> ');

